### DISCUSSION TITLE
Lyrics

### DISCUSSION THREAD

**Discussion by dtl131** [strawberrymusicplayer/strawberry#918] [Ideas]
I think Strawberry could benefit greatly from more control over lyrics. For example, allow the user to see what is available from each lyrics provider and select a provider for each song. Too often, lyrics are missing, or the lyrics are for the correct artist, but the wrong song. Even worse, the lyrics may have been "hijacked" by a hacker and are completely inaccurate.

Thank You

**Comment by s38b35M5** [comment]
Saw the back and forth on your [issue](https://github.com/strawberrymusicplayer/strawberry/issues/996), right as I was checking to see if my FR had been addressed too. I would love if Strawberry could scrape lyrics for all tracks and save to the song file. I have an external mobile device that the player reads lyrics from the file. I wish I could code the change, since the function for the lyric search already exists, but alas, I haven't the experience or know-how. 

"Good enough" will have to do.

**Comment by trincowski** [comment]
This would be good. I see that many times the lyrics are completely unrelated to the song which is playing...

**Comment by jonaski** [comment]
The latest releases have lrclib lyrics provider which has better search based on song artist/title and track length which gives better results.
